### ISSUE TITLE
CheckFatigue issue Fleet moral checking fail

### ISSUE THREAD

**Issue by chicken-eater-bay** [mrmin123/kancolle-auto#359]
*Please read the FAQ/Common Issues section of the README before opening a ticket. Please remove sections of this template you do not find relevant when creating your ticket.*

#### Environment details

* kancolle-auto version (date on first line of your CHANGELOG.md):
* Viewer/browser: KCV
* Operating system: Windows 7 64 bit

kancolle-auto, latest kancolle-auto-master build

#### Issue
I have tested slot 1,2,3,6 and replicated 2 times with red face moral, and kancolle  auto passed the check as moral good.

config:
CheckFatigue = True

**Comment by chicken-eater-bay** [comment]
However Ship(s) with medium fatigue found worked.

![2017-04-15 02_29_46-kancolleviewer](https://cloud.githubusercontent.com/assets/22205034/25052337/ed61a47e-2183-11e7-902f-e38cf55553a6.png)

**Comment by mrmin123** [comment]
@matsumotokenji can you find this line in `kancolle_auto.py`:

```python
# debug_find('dmg_critical.png', 'Chrome', 0.75)  # For debugging purposes only!
```

and replace it with this?:

```python
debug_find('fatigue_high.png', 'Chrome', 0.9)  # For debugging purposes only!
```

(replace 'Chrome' with whatever program you're using, if needed)

Fatigue high should match at 0.98 or above. If they're matching at anything below, please report the values here.

edit: derp, to be clear, make the code changes above, navigate to the pre-sortie screen where you see the fatigue states in-game, then run kancolle-auto. It'll move the mouse over each match, and spit out details for each match in the console.

**Comment by chicken-eater-bay** [comment]
[log] App.focus:  [8480:Chrome]


+  Sikuli match object for 'fatigue_high.png' in window 'Chrome'
+    with minimum similarity of 0.9:
M[1595,300 11x13]@S(S(0)[0,0 1920x1080]) S:0.97 C:1600,306 [-1 msec]
M[1595,450 11x13]@S(S(0)[0,0 1920x1080]) S:0.97 C:1600,456 [-1 msec]

**Comment by matsumotokenji** [comment]
and i don't know if this is normal, before i resupply my fleet:

+  Sikuli match object for 'dmg_critical.png' in window 'Chrome'
+    with minimum similarity of 0.9:
No matches!


[info] Exit code: 0

![2017-04-15 03_03_27-kancolleviewer](https://cloud.githubusercontent.com/assets/22205034/25053184/572ff974-2188-11e7-82db-1e393e5439e3.png)

but after resupply things went ok.
+  Sikuli match object for 'dmg_critical.png' in window 'Chrome'
+    with minimum similarity of 0.9:
M[1574,351 13x24]@S(S(0)[0,0 1920x1080]) S:1.00 C:1580,363 [-1 msec]
M[1574,401 13x24]@S(S(0)[0,0 1920x1080]) S:1.00 C:1580,413 [-1 msec]
M[1574,451 13x24]@S(S(0)[0,0 1920x1080]) S:1.00 C:1580,463 [-1 msec]
M[1574,251 13x24]@S(S(0)[0,0 1920x1080]) S:1.00 C:1580,263 [-1 msec]
M[1574,301 13x24]@S(S(0)[0,0 1920x1080]) S:1.00 C:1580,313 [-1 msec]

**Comment by chicken-eater-bay** [comment]
P/S: == deleted a post found out damage threshold is set to 0.7 ==

**Comment by mrmin123** [comment]
Right, hopefully the damage matching worked after setting it to 0.7.

For the fatigue issue you could modify `FATIGUE_SIMILARITY` in `combat.py` to 0.97 or even 0.96 and that should resolve your issue. When I can, I'll also do some additional testing to see if it's safe to bump the threshold down in `master`.
